docs/design/base-image.md

```diff
@@ -1,14 +1,14 @@
 # The Base Image
 
-The ["base" image][base image] is a small-ish Docker image for running
+The ["base" image][base image] is a small-ish container image for running
 nested containers, systemd, and kubernetes components.
 
 To do this we need to set up an environment that will meet the CRI 
 (currently just docker) and systemd's particular needs. Documentation for each
 step we take is inline to the image's [Dockerfile][dockerfile]),
 but essentially:
 
-- we preinstall tools / packages expected by systemd / Docker / Kubernetes other
+- we preinstall tools / packages expected by systemd / Docker | Podman / Kubernetes other
```

Comment:
this is not true at the moment. this image ships docker for now. it does not ship podman